site/posts/numpy-faq.ipynb

```diff
@@ -5,8 +5,57 @@
    "id": "printable-testing",
    "metadata": {},
    "source": [
-    "Write your post here."
+    "Here is a compilation of questions and issues that arose during the [Numpy](https://colab.research.google.com/drive/1zcjaGXofLzUXkQzJKnANE9Z3yFUt0q7E?usp=sharing) session of the Python Tutorial Seminar Series. \n",
+    "\n",
+    "The live video recording of this content can be found [here](https://www.youtube.com/watch?v=kstc-6uz7AQ&t=10s).\n",
+    "\n",
+    "------"
    ]
+  },
+  {
+   "source": [
+    "**Q.** Is it possible to provide a “step” instead of a number of elements when using `numpy.linspace` to create an array?\n",
+    "\n",
+    "**A.** The step size is derived from the input arguments `start`, `stop`, and `number` with `numpy.linspace`. If you instead want to supply the step size and derive the number of steps, you would instead use the function `numpy.arange` that takes the input arguments `start`, `stop`, and `number`. `numpy.arange()` differs from Python’s built-in `range()` function which only supports integers and produces a `list` rather than an `array`.\n",
+    "\n",
+    "-----\n",
+    "<!-- TEASER_END -->"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** Are there Python methods that mutate the object, or does they always generate a new object? If so, are there any good heuristics for knowing which methods change their objects and which generate changed copies? \n",
+    "\n",
+    "**A.** Yes there are Python functions that mutate the object. The inplace operations are usually defined at the library/package level. Checking the documentation is the safest way to confirm how the library or package behaves.\n",
```

Comment:
What is meant here by "at the library/package level"?  I would interpret that statement to mean functions that look like `numpy.function(myarray)`.  And I know that `numpy.sin` and `numpy.cos` and others are *not* in-place functions, so I'm confused by this.  Maybe this can be clarified.

Reply:
I removed that sentence. I think telling people to check the documentation is what was meant.